bundles/org.openhab.binding.insteon/src/main/resources/msg_definitions.xml

```diff
@@ -1,410 +1,399 @@
 <xml>
-	<!--
-	//    Please keep messages ordered by increasing command number!
-	//
-	// The header is not an official Insteon concept. The boundary
-	// between header and message is the point where enough information
-	// is available to determine the full length of the incoming message.
-	// Sometimes that's just two bytes, sometimes 6 or 9.
-	//
-	->
+	<!-- // Please keep messages ordered by increasing command number! // // The header is not an official Insteon concept.
```

Comment:
This didn't format correctly

Reply:
Maybe we should exclude comments in XML from formatting.

Reply:
I hope that will also stop it from messing up license headers on XML files. We saw that happen when applying it in openhab-core.

Reply:
I found that https://github.com/openhab/static-code-analysis/pull/375 fixes the license formatting issues in SAT 0.10.0-SNAPSHOT though the plugin configuration was missing so I've fixed that with https://github.com/openhab/openhab-core/pull/1442.

We probably want to apply something similar for all XML files, so you could test that by adding

`formatCommentJoinLines=false`

if that doesn't help, all XML comment formatting can be disabled with:

`formatCommentText=false`

Reply:
This appears to be fixed in the latest commit.